docs/source/configuration/traffic-shaping.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Traffic shaping
+description: Configuring traffic shaping
+---
+
+import { Link } from "gatsby";
+
+> ⚠️ Apollo Router support for traffic shaping is currently experimental.
+
+The Apollo Router provides experimental support for traffic shaping.
+
+Currently features are limited, but are expected to grow over time:
+
+* [x] **Sub-query deduplication** - Identical in flight sub-queries are compressed into a single request.
+* [ ] Rate limiting
```

Comment:
let's maybe not commit on rate limiting right now, it might come much later depending on how we specify it

Reply:
I removed the entire section about future functionality.